### DISCUSSION TITLE
Gabor embedded in dynamic noise

### DISCUSSION THREAD

**Discussion by giuliaccc222** [jspsych/jsPsych#1430] [Q&A]
Hi, I'm working on an experiment where I should present gabors varying in contrast embedded in dynamic noise. I was thinking to use the rdk plugin to potentially generate noise and display gabors on top of it, but I am not sure whether that's possible (I guess it would mean to combine two plugins/present a stimulus generated by a plugin while maintaining the other in the background). Alternatively, I was also wondering whether there's already a plugin that allows to do that (generate and control both gabors and noise at the same time)?

Many thanks for your time!

**Comment by kurokida** [comment]
@giuliaccc222 , [my plugin supported the presentation of gabor patches](https://jspsychophysics.hes.kyushu-u.ac.jp/gabor.html). But there are still accuracy and display calibration issues. Moreover, may not be able to present as many patches as noise.
